ansible/playbooks/roles/elasticsearch_curator/tasks/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Include installation task
-  include_tasks: install-es-curator-{{ ansible_os_family }}.yml
+  include_tasks: install-ops-curator-{{ ansible_os_family }}.yml
```

Comment:
Exactly. That's what I've pointed out in the [docs](https://github.com/romsok24/epiphany/blob/feature/migr-ODFE-OpenSearch-2870/docs/home/howto/UPGRADE.md#:~:text=Migration%20of%20Elasticsearch%20Curator%20is%20not%20supported), however I didn't want to take this decision by myself only.
Are You ok to create an Issue for that decision?

Reply:
We had related issue #2423 but it was closed because of switching to OpenSearch. Don't know how much `curator-opensearch` differs from Elasticseach Curator and whether it can handle [this use case](https://github.com/epiphany-platform/epiphany/issues/2423#issuecomment-932107778).

IMO we could create spike to check whether [ISM](https://opensearch.org/docs/latest/im-plugin/ism/index/) is able to handle the use case first. And if yes, then we can implement ISM and remove Curator. Otherwise, I would spike `curator-opensearch` as second choice option.

What do you think @seriva?

Reply:
A [spike ](https://github.com/epiphany-platform/epiphany/issues/3003)created.